apps/examples/histogrammer.cpp

```diff
@@ -20,15 +21,31 @@ void CheckArgs(int argc, char **argv) {
 }
 
 int main(int argc, char **argv) {
-  CheckArgs(argc, argv);
-  ConfigManager::Initialize(argv[1]);
+  auto args = make_unique<ArgsManager>(argc, argv);
+  // check if optional value "config" is present
```

Comment:
you can remove this comment, it was just to get the lines of code from copilot ;)